front-end/src/pages/LoginPage.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useState } from 'react';
+import useForm from '../hooks/useForm';
+
+export default function Login() {
+  const [isValid, setIsValid] = useState(true);
+  const { values, handleChange } = useForm({ email: '', password: '' });
+  const validade = (loginInfo) => {
```

Comment:
Acho que poderia chamar `validate`, assim fica padronizado em inglês. O que acha?
```suggestion
  const validate = (loginInfo) => {
```